client/src/partials/templates/modals/findEntity.modal.html

```diff
@@ -0,0 +1,74 @@
+<div class="">
+  <div class="panel-heading">
+    <h4>
+      <i class="glyphicon glyphicon-search"></i>
+      {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
+      <span
+        ng-if="$ctrl.result.uuid"
+        ng-click="$ctrl.refresh()"
+        class="btn btn-default pull-right">
+        <i class="glyphicon glyphicon-refresh"></i>
+      </span>
+    </h4>
+  </div>
+
+  <div class="panel-body">
+
+    <div class="form-group">
+
+      <div ng-if="!$ctrl.result.uuid" class="input-group">
+
+        <!-- dropdown button -->
+        <div class="input-group-btn" uib-dropdown tabindex=-1>
+          <button class="btn btn-default" type="button" uib-dropdown-toggle>
+            {{ $ctrl.selectedTypeLabel | translate }} <span class="caret"></span>
+          </button>
+          <ul uib-dropdown-menu>
+            <li ng-repeat="type in $ctrl.types">
+              <a href="" ng-click="$ctrl.setType(type)">
+                {{ type.label | translate }}
+              </a>
+            </li>
+          </ul>
+        </div>
+        <!-- /dropdown button -->
+
+        <input type="text" class="form-control"
+          ng-model="$ctrl.entity"
+          placeholder="{{ $ctrl.placeholder | translate }}..."
+          uib-typeahead="entity as entity.text for entity in $ctrl.entities | limitTo: 10"
+          typeahead-on-select="$ctrl.selectEntity($item)"
+          autocomplete="off"
+          autocorrect="off">
+
+      </div>
+
+      <div ng-if="$ctrl.result.uuid" class="bg-success">
+        <h4 class="text-center text-success" style="padding:10px; margin:0px;">
+          <i class="glyphicon glyphicon-user"></i>
+          {{ $ctrl.result.label }}
+        </h4>
+      </div>
+
+    </div>
+  </div>
+
+  <div class="panel-footer">
+    <div class="pull-right">
+      <button type="button" class="btn btn-default"
+        ng-click="$ctrl.cancel()"
+        data-button-cancel>
+        <i class="glyphicon glyphicon-ban-circle"></i>
```

Comment:
I think we should either use icons on all our modal buttons, or none of them.  At the moment we haven't been using icons.

If you think icons on these buttons makes a better user experience, could you create an issue (like #268) to propose them?  This would ensure that the team all uses the same standards.